cmd/plume/specs.go

```diff
@@ -207,6 +207,56 @@ var (
 				},
 			},
 		},
+		"developer": channelSpec{
+			BaseURL: "gs://builds.developer.core-os.net/boards",
+			Boards:  []string{"amd64-usr", "arm64-usr"},
+			Destinations: []storageSpec{storageSpec{
```

Comment:
We don't actually need the artifacts re-uploaded anywhere, so I'd be inclined to drop this.